iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -39,15 +39,22 @@
 
     private boolean throwExceptionIfExtensionNotFound = false;
 
+    private String version;
+
     public ExportOptions() {
     }
 
     public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel, boolean throwExceptionIfExtensionNotFound) {
+        this(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound, null);
```

Comment:
To be discussed: I'm not sure we should allow null values

Reply:
In my opinion, `null` for version would mean the same as "default" version i.e. the most recent version (which is implemented in `NetworkXml`).

The logic here of using `null` for undefined rather than directly a defined value is that if we use `ExportOptions` for an another serialization of IIDM in the future (IIDM-JSON for example), the most recent version of this serialization does not have to be the same as IIDM-XML most recent version. In this case, keeping it undefined and letting the converter itself assert the defaut version is the best solution I think.